storm_tests/generic/Rakefile

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+OBJECTS = %w[
+  circular_queue_test.o
+  generic_tests_main.o
+].freeze
+
+OUTPUT = :generic_tests
+
+# CFLAGS are set in storm_tests.rake
+
+# All functions overridden by tests must explicitly be listed here, to let the linker redirect
+# the function calls to the wrappers instead of the real implementations.
+#
+# Like this: -Wl,--wrap=memory_global_allocate
+LDFLAGS = %w[
+  -m32
+  -L/usr/local/lib
+  -L../../storm/generic
+  -L../../storm/x86
+].freeze
+
+# Yes, the storm_* actually need to be specified this many times.
+# Without it, GNU ld fails to figure out how to link the tests together.
+LIBRARIES = %w[
+  cmocka
+
+  storm_x86
+  storm_generic
+  storm_x86
+  storm_generic
+  storm_x86
+].freeze
```

Comment:
This is one of the more bizarre parts of the whole cmocka/unit testing approach... Could look into a better fix someday, but hey: it works. 😉